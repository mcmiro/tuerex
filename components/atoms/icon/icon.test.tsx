/* eslint-disable */
import * as React from 'react';
import { screen, render } from '@testing-library/react';
import Icon from '.';
import { CheckIcon } from '@heroicons/react/24/outline';

describe('Test icon component', () => {
  it('Should pass when icon is true', () => {
    render(
      React.cloneElement(<Icon icon={CheckIcon} data-testid="icon-test" />)
    );

    const renderIconComponent = screen.queryByTestId('icon-test');
    expect(renderIconComponent).toBeDefined();
    expect(renderIconComponent).toMatchSnapshot();
  });
});
