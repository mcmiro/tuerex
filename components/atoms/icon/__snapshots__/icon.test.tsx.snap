// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test icon component Should pass when icon is greater false 1`] = `null`;

exports[`Test icon component Should pass when icon is greater true 1`] = `null`;

exports[`Test icon component Should pass when icon is true 1`] = `null`;
